index.html

```diff
@@ -5,13 +5,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <meta name="description" content="This is a knowledge based website where you get to ask any question and get the answers you need" />
+    <meta name="description"
+        content="This is a knowledge based website where you get to ask any question and get the answers you need" />
     <meta name="keywords" content="questions,search, comments, opinions" />
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Luckiest+Guy&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Monoton&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Aclonica&display=swap" rel="stylesheet">
```

Comment:
not properly linking to the css and why are you importing so many fonts from google....